src/inc/readytorun.h

```diff
@@ -55,6 +55,7 @@ enum ReadyToRunSectionType
     READYTORUN_SECTION_DEBUG_INFO                   = 105,
     READYTORUN_SECTION_DELAYLOAD_METHODCALL_THUNKS  = 106,
     READYTORUN_SECTION_AVAILABLE_TYPES              = 107,
```

Comment:
The earlier `READYTORUN_SECTION_AVAILABLE_TYPES` is not going to be compatible with the current one because of the hash algorithm changed. Could you please change `107` to be unassigned, and make:
```
READYTORUN_SECTION_AVAILABLE_TYPES              = 108,
READYTORUN_SECTION_INSTANCE_METHOD_ENTRYPOINTS  = 109,
```